plugin-server/src/cdp/consumers/cdp-source-webhooks.consumer.ts

```diff
@@ -16,6 +16,8 @@ import { createAddLogFunction } from '../utils'
 import { createInvocation, createInvocationResult } from '../utils/invocation-utils'
 import { CdpConsumerBase } from './cdp-base.consumer'
 
+const DISALLOWED_HEADERS = ['x-forwarded-for', 'x-forwarded-host', 'x-forwarded-proto', 'x-forwarded-port', 'cookie']
```

Comment:
May want to include the below, to be safe:

- `x-csrftoken`
- `authorization`
- `proxy-authorization`
- `referer`
- `forwarded`
- `x-real-ip`
- `true-client-ip`

This list can change as our infra changes (e.g. if we start using Cloudflare that adds some additional headers). It  would be much preferable to operate off of an allowlist. I don't have a great understanding of the current use case so it's hard to say exactly, but something like this
```ts
const ALLOWED_HEADERS = ['Accept', 'Accept-Encoding', 'Accept-Language', 'Cache-Control', 'Pragma', 'Content-Type', 'Content-Length', 'Content-Encoding', 'Content-Language', 'User-Agent', 'Host', 'Date']
```